<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>stock analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2">
        <item>
         <widget class="QTableWidget" name="SearchTableWidget">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SearchPushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLineEdit" name="FilterLineEdit"/>
             </item>
             <item>
              <widget class="QListWidget" name="StockListWidget">
               <property name="modelColumn">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParament_List">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionPE"/>
    <addaction name="actionPB"/>
    <addaction name="actionTime"/>
    <addaction name="actiontotal"/>
    <addaction name="actionPrice"/>
    <addaction name="actionindustry"/>
    <addaction name="actionarea"/>
    <addaction name="actionoutstanding"/>
    <addaction name="actiontotalAssets"/>
    <addaction name="actionliquidAssets"/>
    <addaction name="actionfixedAssets"/>
    <addaction name="actionreserved"/>
    <addaction name="actionreservedPerShare"/>
    <addaction name="actionesp"/>
    <addaction name="actiontimeToMarket"/>
    <addaction name="actionundp"/>
    <addaction name="actionperundp"/>
    <addaction name="actionrev"/>
    <addaction name="actionprofit"/>
    <addaction name="actiongpr"/>
    <addaction name="actionnpr"/>
    <addaction name="actionholders"/>
   </widget>
   <addaction name="menuParament_List"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PE</string>
   </property>
  </action>
  <action name="actionPB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PB</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </action>
  <action name="actiontotal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>totals</string>
   </property>
  </action>
  <action name="actionPrice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </action>
  <action name="actionindustry">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>industry</string>
   </property>
  </action>
  <action name="actionarea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>area</string>
   </property>
  </action>
  <action name="actionoutstanding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>outstanding</string>
   </property>
  </action>
  <action name="actiontotalAssets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>totalAssets</string>
   </property>
  </action>
  <action name="actionliquidAssets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>liquidAssets</string>
   </property>
  </action>
  <action name="actionfixedAssets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>fixedAssets</string>
   </property>
  </action>
  <action name="actionreserved">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>reserved</string>
   </property>
  </action>
  <action name="actionreservedPerShare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>reservedPerShare</string>
   </property>
  </action>
  <action name="actionesp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>esp</string>
   </property>
  </action>
  <action name="actiontimeToMarket">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>timeToMarket</string>
   </property>
  </action>
  <action name="actionundp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>undp</string>
   </property>
  </action>
  <action name="actionperundp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>perundp</string>
   </property>
  </action>
  <action name="actionrev">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>rev</string>
   </property>
  </action>
  <action name="actionprofit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>profit</string>
   </property>
  </action>
  <action name="actiongpr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>gpr</string>
   </property>
  </action>
  <action name="actionnpr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>npr</string>
   </property>
  </action>
  <action name="actionholders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>holders</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
